### PR TITLE
Re-enable kinesis test

### PR DESCRIPTION
**Pull request by KKcorps**

### CODE REVIEW

File: pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/RealtimeKinesisIntegrationTest.java [hunk 1]
```diff
@@ -83,8 +85,8 @@
 import software.amazon.awssdk.utils.AttributeMap;
 
 
-@LocalstackDockerProperties(services = {"kinesis"})
-@Test(enabled = false)
+@LocalstackDockerProperties(services = {ServiceName.KINESIS}, portEdge = "4591", environmentVariableProvider = RealtimeKinesisIntegrationTest.HostnameEnvProvider.class, imageTag = "0.13.0.10")
```
Comment (xiangfu0) [inline pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/RealtimeKinesisIntegrationTest.java:88]: can this port be dynamic?

Comment (xiangfu0) [reply pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/RealtimeKinesisIntegrationTest.java:88]: Also need fix linter, line too long

Comment (KKcorps) [reply pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/RealtimeKinesisIntegrationTest.java:88]: Hi, @xiangfu0 don't review this PR yet. it is still WIP. this kinesis issue is not reproducible in my local and any other VMs I created. So I raised this PR just to test out what is going wrong.
Will fix linting once I have a working solution.

**Comment by KKcorps** [comment]
@xiangfu0 @npawar The test is passing now. Their seems to an issue with latest localstack docker. It was always pulling the image with `latest` tag. I have now specified the tag with which the test was originally created.

**Review by npawar — APPROVED** [review]
Good catch! lgtm

**Review by siddharthteotia — APPROVED** [review]

**Pull request merged**